x-pack/plugins/security_solution/public/resolver/types.ts

```diff
@@ -576,7 +673,15 @@ export interface IsometricTaxiLayout {
   /**
    * defines the aria levels for nodes.
    */
-  ariaLevels: Map<SafeResolverEvent, number>;
+  ariaLevels: Map<ResolverNode, number>;
+}
+
+/**
+ * Defines the type for bounding a search by a time box.
+ */
+export interface TimeRange {
+  from: Date;
```

Comment:
can change later, but I think it would make sense to have this type mirror or even re-use what exists in security_solution and just have these be strings.

Reply:
👍  yeah sounds good to me.